README.md

```diff
@@ -186,6 +186,40 @@ In dark mode!
 ![Job page](img/j3.png)
 
 
+## Instructions to run
+```
+$ git clone https://github.com/FahimFBA/StudentBook.git
+$ cd StudentBook
+$ npm install
+$ npm run dev
+```
+
```

Comment:
You have still kept the instructions to clone, installing dependencies and so on.